tools/PI/DevHome.PI/Helpers/WindowHelper.cs

```diff
@@ -590,12 +590,13 @@ internal static async void ShowTimedMessageDialog(FrameworkElement frameworkElem
 
         try
         {
-            await ContentDialog.ShowAsync();
-            timer.Start();
             ContentDialog.Closed += (s, e) =>
             {
+                timer.Stop();
                 ContentDialog = null;
             };
+            _ = ContentDialog.ShowAsync();
+            timer.Start();

```

Comment:
Alternative suggestion. Not exactly sure which part of the code may throw an exception, but if we're not awaiting, an exception from `ShowAsync` will not be captured in the catch block.
```suggestion
            ContentDialog.Opened += (s, e) => timer.Start();
            ContentDialog.Closed += (s, e) =>
            {
                timer.Stop();
                ContentDialog = null;
            };
            await ContentDialog.ShowAsync();
```

Reply:
Thanks for the suggestion! Fixed.